_guides/contextual_bandits.md

```diff
@@ -8,108 +8,114 @@ show_on_learn: false
 layout: tutorial
 body_class: tutorial
 resource_icon: /svg/resources/guide.svg
----
```

Comment:
Need to keep the dash, this is required for the framework we are using.

Reply:
I don't see line 11 on my version. It's a blank line. So I add one dash is all?